pkg/util/util.go

```diff
@@ -16,6 +16,15 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// CutString makes sure the string length doesn't exceed 255, which is usually the maximum string length in OpenStack.
+func CutString(original string) string {
```

Comment:
If this lands on utils you got to either name this something like `CutStringTo255()` or allow passing `255` as a parameter.

Reply:
Renamed to `CutStringTo255()`.